packages/braid-design-system/src/lib/components/Toggle/Toggle.tsx

```diff
@@ -43,6 +44,7 @@ export const Toggle = forwardRef<HTMLInputElement, ToggleProps>(
       onChange,
       label,
       align = 'left',
+      togglePosition = align === 'left' ? 'leading' : 'trailing',
```

Comment:
Not sure about using a ternary as a default value like this, and this may be the reason for the Playroom note below.

Given it is an override for the layout (previously controlled solely by `align`), knowing if a value has been passed in explicitly versus a default would be my recommended approach.

Reply:
I've moved the default setting logic into the component itself, but not 100% clear how much simpler this approach is. The Playroom issue is actually present in prod with the `algin` prop as well.